picard/ui/newuserdialog.py

```diff
@@ -0,0 +1,73 @@
+# -*- coding: utf-8 -*-
+#
+# Picard, the next-generation MusicBrainz tagger
+#
+# Copyright (C) 2023 Bob Swift
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+
+
+from PyQt5 import QtWidgets
+
+from picard.const import PICARD_URLS
+
+
+class NewUserDialog():
+
+    def __init__(self):
+
+        self.DIALOG_TEXT = _(
+            (
+                "<p>"
+                "MusicBrainz Picard is an extremely powerful cross-platform music file tagger.  "
+                "In addition to downloading metadata from the MusicBrainz database and updating the tags in your music files, "
```

Comment:
I think an additional sentence would be useful here to explain that the primary purpose of Picard is to tag music files with metadata held in Musicbrainz (overwriting most if not all existing tags) and optionally renaming files based on this same metadata - and also that the majority of usage is to tag albums rather than individual files - and that if this is not your goal, then Picard may not be the right tagger for your needs.

Reply:
Also, I think that this should be a standard disclaimer that is also included word for word in the introductory web pages of the documentation site.

Reply:
I think the text is too long now, and people are unlikely to read it.

![image](https://github.com/metabrainz/picard/assets/151042/044e6d80-c8e6-4c41-9c6c-4751d38d918b)

I suggest we keep it as short as possible, being very clear about risks, but that's all.

1) Read the documentation
2) Work on copies and in small batches
3) Use at your own risk

What do you think?

Reply:
@zas Yes - I think you are right. But unfortunately the alternative would be to simplify it and include a link to the same or an even longer explanation which would then be even less likely to be read. Here is my attempt at making this more punchy:

>                 NEW USERS - READ THIS BEFORE USING PICARD
>
> Picard is designed to download music meta-data and cover art from MusicBrainz (a massive online database of information about music releases) and then use it to overwrite any existing tags and rename and move the files into a structured "library". It is best used with albums, though with careful configuration it can be used with individual tracks. If this is not your intent, then this may not be the best tool for you. 
>
> If this is what you want to achieve, then Picard can be a very powerful tool for you, but if you configure its options incorrectly, you can create havoc with your music collection.
>
> Thousands of users around the world successfully use Picard to tag and structure their music collections, and so that you also can use Picard successfully to tag your music collection we recommend that you:
> 
> 1. Read the HELP documentation - press F1 or use the Help menu. Use it to understand how to use Picard to match your music files against releases in the MusicBrainz database.
> 2. Configure your file naming script carefully.
> 3. Work on copies of your music files and in small batches until you are fully confident that your music files will be handled the way you want them too. Once Picard has updated a file, there is no way to undo the changes.
>
> ✅ Show this message again the next time you start Picard
>
> Picard is open source software written by volunteers and as such is provided as-is and with no warranty - you use Picard at your own risk.

Reply:
@Sophist-UK your suggestion is even longer, and as is, I'm pretty sure new users will press Ok and don't read.
If people aren't interested in details and don't follow the link or follow it and don't read, well, that's their problem not ours.

We need to provide concise information about risks for audio files to new users. The message has to be short and clear, providing a link is just for 1% of users that will bother clicking it, and for 1% of those that will actually read the page.

What we need instead is a message 80% of new users will read, and as such it has to be very very short and impactful (like a good ad).

To me what you suggest would perfectly fit the linked document, but not the dialog box.

@rdswift @phw what do you think?

Reply:
@zas - Yes, I know mine was actually about 25% longer, but then again it actually says more and in a more punchy way - and I am sure that it can be cut down a bit. Here is a short, blunt alternative:

>            NEW USERS - READ THIS BEFORE USING PICARD
>
> Picard is a very flexible music tagging tool, using MusicBrainz metadata for tags and file naming, but if you don't understand how it works, what its limitations are and how to configure it you can easily screw up your music library.
>
> **We therefore STRONGLY recommend that you:**
> 1. Read the [Picard documentation at https://picard-docs.musicbrainz.org/](https://picard-docs.musicbrainz.org/) before you use this tool on your music collection.**
> 2. Work on copies of your music files and in small batches until you are fully confident that your music files will be handled the way you want them too. Once Picard has updated a file, there is no way to undo the changes.
>
> ✅ Show this message again the next time you start Picard
>
> Picard is open source software written by volunteers and as such is provided as-is and with no warranty - you use Picard at your own risk.

Reply:
I like the shortened version, and have updated the PR accordingly.  I also added the number of files that will be processed to the file save warning dialog.